src-ghc/Pact/GasModel/GasTests.hs

```diff
@@ -1506,6 +1507,16 @@ makeListTests = defGasUnitTests allExprs
 
     allExprs = NEL.map (createPactExpr makeListExpr) sizesExpr
 
+enumerateTests :: NativeDefName -> GasUnitTests
```

Comment:
Great job adding gas tests! (I haven't had to do one yet 😅 ) However, `makeListTests` just above is more useful for gas. Compare its output to that of `enumerate` in the golden output above and you'll see gas varying significantly, as the gradual exponential cost model is flat for small lists.

In other words, consider explicit examples to exercise a broader range of sizes or somehow use `sizesExpr` from Linda's code in `makeListTests` at L1508.